addons/website_event/views/event_event_views.xml

```diff
@@ -29,20 +29,22 @@
                 <attribute name="context">{'default_website_id': website_id}</attribute>
                 <attribute name="groups">website.group_multi_website</attribute>
             </xpath>
+            <xpath expr="//div[@name='button_box']/button[hasclass('oe_stat_button')]" position="before">
+                <field name="is_published" widget="website_redirect_button"/>
+            </xpath>
             <div name="button_box" position="inside">
                 <field name="website_url" invisible="1"/>
-                <field name="is_published" widget="website_redirect_button"/>
             </div>
             <xpath expr="//div[hasclass('oe_title')]" position="after">
-                <div name="event_menu_configuration" groups="base.group_no_one">
-                    <label for="website_menu" string="Website Submenu"/>
+                <div name="event_menu_configuration" groups="base.group_no_one" class="mb-3">
                     <field name="website_menu"/>
+                    <label class="me-5" name="website_menu" for="website_menu" string="Website Submenu"/>
                     <!-- hidden sub-menus, they are triggered all at once based on "website_menu" -->
                     <field name="introduction_menu" invisible="1"/>
                     <field name="location_menu" invisible="1"/>
                     <field name="register_menu" invisible="1"/>
-                    <label for="community_menu" string="Community" invisible="1"/>
                     <field name="community_menu" invisible="1"/>
+                    <label class="me-5" for="community_menu" string="Community" invisible="1"/>
```

Comment:
@khpa-odoo It always adds one space in the last so can we add CSS class in the <field> OR do you have any other suggestion?

Reply:
I could remove the `me-5` from the last 'community_menu' but it won't be appropriate for future use.
Hence, added `ms-5` to the field as you suggested :+1: